common/src/main/resources/data/farm_and_charm/crafting_bowl/corn_dough.json

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "farm_and_charm:crafting_bowl",
+  "ingredients": [
+    {
+      "item": "cornexpansion:corn_flour"
+    },
+    {
+      "item": "farm_and_charm:corn"
```

Comment:
I'm not sure about using raw corn for the dough, isn't the flour enough ?

Reply:
Probably yes. I'll revise tonight